### ISSUE TITLE
Make performance testing less confusing: write foundational docs (ready with tooling)

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/perf-frontend-issues#89]
We have no defacto documentation on how to do performance testing. This leads to confusion around:
- Which build variant do I use?
- How do I use the profiler? When should I not trust the profiler?
- How do I figure out if my patch impacts cold startup? How do I debug it if it does?
- What's the difference between fenix-nightly and fennec-nightly variants? Which do I test on?
- How do I get help?

In fact, we can't even be certain that we're on the same page within the team. We should write some documentation to clarify.

---

Triage: waiting on the future of FNPRMS to decide what to do with this documentation.

In the short term, we can consolidate the few docs we have into a single guide so folks can continuously add things as they notice them. I (mcomella) volunteered to maintain the guide to ensure it stays clear as more data is added.

**Comment by mcomella** [comment]
We don't really understand the need here so closing in favor of a simpler splash page solution: https://github.com/mozilla-mobile/perf-frontend-issues/issues/105
